python/procstar/agent/conn.py

```diff
@@ -156,20 +158,20 @@ def conn_id(self):
 
     @property
     def open(self):
-        return self.ws.open
+        return self.ws.state is websockets.protocol.State.OPEN
```

Comment:
nit: I think it's better to use `==` for enum comparisons
```suggestion
        return self.ws.state == websockets.protocol.State.OPEN
```